### ISSUE TITLE
clarify details related to ancillary_variables

### ISSUE THREAD

**Issue by jenseva** [OceanGlidersCommunity/OG-format-user-manual#239]
moderator: @OceanGlidersCommunity/format-maintainers

**Is your feature request related to a problem? Please describe.**

:ancillary_variables: Are you planning to allow flexibility in ancillary var name? This is not clear in the manual. Right now the qc var names look rigid. 

In my files I can often use the same qc var for two variables. For example, gps_latitude and gps_longitude vars have the same qc and flags so only one var is needed (gps_qc). I assume this is acceptable to OG-1.0, but not 100% clear.

**Is this related to a specific platform models**

I expect this is common practice for many types of gliders.

**Describe the solution you'd like**

Clarify in the documentation that flexibility in ancillary variable name is allowed and that one qc var can be used for multiple variables.

I can make a PR for this if there is agreement.

**Comment by castelao** [comment]
Good point. I agree that `ancillary_variables` should be flexible. Somehow in the documentation it would be worth saying that it is strongly recommended to use when relevant.

**Comment by emmerbodc** [comment]
I'm keen to keep the <PARAM>_QC fixed as it is using the OG1 vocabulary for <PARAM>

**Comment by jenseva** [comment]
Yes, the _qc remains the same, but it would be good to have flexibility in the text before that. 

Where for instance: gps_latitude and gps_longitude vars have the same flag variable named gps_qc. The mapping to the qc vars is always clearly listed in the ancillary_variables attribute which is used by both humans and machines easily.

Without this ability the OG files will be larger, requiring replication of qc variables. The impact of this is doubles when using primary and secondary flag variables. I recommend this approach because it is both user-friendly and space-friendly.

**Comment by vturpin** [comment]
Is this only applicable for locations ?

**Comment by jenseva** [comment]
This would apply to any situation where one qc variable represents the qc for multiple PARAM variables.  

A statement along the lines of, "In cases where one qc variable applies to multiple variables, the portion of qc variable name preceding _qc can be determined by the provider. For an example of this scenario, see the gps variables in the Spray sp028 example file."

**Comment by vturpin** [comment]
I see the point. This flexibility seems fine to me.
